### PR TITLE
Logging guild activity

### LINKED ISSUE #130: Add logging guild activity

### PR DESCRIPTION
**Pull request by Niewidzialny84**
Closes #130

### CODE REVIEW

File: DiscordBot.Core/Providers/ChannelDataProvider.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        private Dictionary<ulong, ulong> channelDict;
+
+        public ChannelDataProvider()
+        {
+            channelDict = new Dictionary<ulong, ulong>();
+        }
+
+        public ulong? getChannel(ulong guildId)
+        {
+            if(channelDict.ContainsKey(guildId))
```
Comment (seker212) [inline DiscordBot.Core/Providers/ChannelDataProvider.cs:20]: `public ulong? getChannel(ulong guildId) => channelDict.ContainsKey(guildId) ? channelDict[guildId] : null;`

File: DiscordBot.ActivityLogging/VoiceChannelActivity.cs [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            else
+            {
+                return null;
+            }
+        }
+
+        private async Task SendLogsToChannel(String mention, SocketTextChannel textChannel, String action)
+        {
+            //TODO timezone handling
+            //var datetime;
```
Comment (seker212) [inline DiscordBot.ActivityLogging/VoiceChannelActivity.cs:50]: remove unused code

**Review by seker212 — APPROVED** [review]
Approved with suggestions

File: DiscordBot.Core/Providers/ChannelDataProvider.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                return channelDict[guildId];
+            }
+            return null;
+        }
+
+        public void setChannel(ulong guildId, ulong channelId)
```
Comment (seker212) [inline DiscordBot.Core/Providers/ChannelDataProvider.cs:27]: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

File: DiscordBot.ActivityLogging/VoiceChannelActivity.cs [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        }
+
+        public Task Execute(SocketUser socketUser, SocketVoiceState beforeVoiceState, SocketVoiceState afterVoiceState)
+            => Task.Run(async () =>
+            {
+                var userMention = socketUser.Mention;
+                var textChannel = await GetTextChannel(beforeVoiceState.VoiceChannel, afterVoiceState.VoiceChannel);
+
+                if(textChannel == null)
+                {
```
[the remark this answers is not in the document]
Comment (Niewidzialny84) [reply DiscordBot.ActivityLogging/VoiceChannelActivity.cs:71]: How about no.

**Pull request merged**